20200326/MINUTES.md

```diff
@@ -0,0 +1,106 @@
+	     dependent and this has to be changed.
+        - [ ] MaxIV beats configuration status: update from  Alejandro's work.
+	     There is no repository with beats configuration, but they analized it.
+	     In Alba were using Logstash and spliting the entries, filebeats
+	     comes with this possibility, it splits by itself using modules. Performance and readibility
+	     are being investigated and compared. Perhaps the parsing of log lines could be done in
+	     a filebeat sardana module.
+	     In Alba also analyse metricbeat: https://www.elastic.co/beats/metricbeat
+        - [ ] Add information on how to access debug information of libtango9-dbgsym (seems like some information is not accessible)
+	     No update
+        - [ ] Users can break Sardana by messing with configuration in Tango DB, diagnostic script should be created to check that
+	     No update
+        - [ ] Few words on Python3 in Newsletter: status update
+	     No update
+        - [ ] Configure CI on buster (problems with test suite on Python 3.6): status update
+	     No update
+3. Overview of current developments / PR distribution
+        - [ ] 2nd Ctrl+C abort the macro: status update
+	     No update
+	- [ ] Zibi will review the last debug information from Thorsten
+	     Zibi analysed the last logs, PR for dumping status information of the counter/timers is done.
+	     See discussion in PR#1308
+	- [ ] Juliano will share the code of the proxy/mirror controller that they use
+	     Aureo will provide the code after the meeting.
+    * How Tango issues affects Sardana?
+        * https://github.com/tango-controls/pytango/issues/292 and https://github.com/tango-controls/cppTango/issues/686
+	 Zibi advanced in the workarounds in tango for that. The bug affects the 4th beamline migrating
+	 to python3. It would be imposible to migrate it to python3 if this is not solved.
+	 Three workarounds are evaluated (PRs to Taurus). One seems to solve the problem.
+	 cpptango has a PR to solve the issue and will be evaluated.
+	 Could be that the PRs for Taurus are not needed any more.
+        * https://github.com/tango-controls/pytango/issues/307 and https://github.com/sardana-org/sardana/pull/1298
+	 Jairo is working on this, it is not easy to solve in pytango, probably it has to be done in cpptango and
+	 see how logs are constructed.
+    * Several PMTV and macroexecutor/sequencer improvement PRs
+        No update
+    * showscan online improvements
+        There are two PRs related. One changing the way the trends are updated (it was done periodically and now is based
+	on events, this PR is integrated. In other one Tiago works in data inspection for showscan online (work in progress).
+    * Jan20 release:
+	Some PRs have been integrated but there are still 8 PRs to be reviewed and merged. They have been assigned.
+    * sardana-org/sardana#1315
+         A decision is postponed and will be done in the PR
```

Comment:
Here we said that we will all think about it and reply in the issue. PR will be created afterwards when we agree on what to remove and what will stay.